Classes/Job/StaticMethodCallJob.php

```diff
@@ -94,8 +94,8 @@ public function getLabel()
     {
         $arguments = [];
         foreach($this->arguments as $argumentValue) {
-            if (TypeHandling::isSimpleType($argumentValue)) {
-                $arguments[] = $argumentValue;
+            if (TypeHandling::isSimpleType(gettype($argumentValue))) {
+                $arguments[] = gettype($argumentValue);
             } else {
                 $arguments[] = '[' . gettype($argumentValue) . ']';
```

Comment:
Bonus points for using `TypeHandling::getTypeForValue($argumentValue)` here, so it's not only `object` but a class name…